### DISCUSSION TITLE
Looking for info on how to set equalizer via NVS editor

### DISCUSSION THREAD

**Discussion by HiFiJ** [sle118/squeezelite-esp32#404] [Q&A]
Hello everyone,

thanks so much for this cool project,  I jumped aboard using it on an Esparagus Hifi MediaLink! I received it a couple of days ago and quickly found my way around and so far it is working really nice!

I just wanted to get some clues about the equalizer use. I understand that I have to use the 16 Bit firmware and flashed this (is there a way to confirm the running variant from the WebUI?).
Also I found I have to use the squezzeesp32 plugin for LMS, which I installed. I also get the esp32 settings **entry**  in the Player Tab, but unfortunately the page itself remains empty (no equalizer or other settings are shown). Perhaps the plugin does not play nicely with LMS 8.4.1 (?).

But I also found that the NVS editor on the WebUI exposes the euqalizer settings as "0,0,0,0...". I hope to be able to just set the settings here, without using the LMS server interface.

Could someone help me out with the supported values for the parameters and how to set them properly?

Thank you and best regards,

Joost

**Comment by HiFiJ** [comment]
After a couple of restarts the settings page for ESP32 on LMS showed up, so I am (successfully) tuning equalizer settings there now.

**Comment by sle118** [comment]
@HiFiJ equalizer is exclusively controlled by LMS. Changing it in the NVS will not persist after the system synchronizes with the server. The squeezelite-esp32 plugin has to be installed and kept up to date on the server as well.

**Reply by DavidM42** [reply]
So do I understand it correctly that to have any eq you have to keep it connected to lms with updated esp32 plugin?
Even if I set the eq via lms, then stop my lms server and use the device standalone the eq will not be applied anymore?
Related to that the LMS plugin from the plugin selector does not work in lms 9 right now
